autoload/coc_explorer.vim

```diff
@@ -175,6 +175,7 @@ function! coc_explorer#init_buf(is_border)
               \ noswapfile noundofile
               \ nomodeline
               \ signcolumn=no
+              \ nocursorline
```

Comment:
@DanilaMihailov I think `nocursorline` and `nocursorcolumn` should stay on the same line because their functions are similar

Reply:
makes sense :) done.